README.md

```diff
@@ -18,97 +18,76 @@ It's easy as:
 git clone https://github.com/theiconic/docker-nginx-akeneo.git
```

Comment:
there is an extra space in "## # Running Service Clusters"

## # Running Service Clusters